packages/core/src/node/cluster/master-process.ts

```diff
@@ -49,14 +59,15 @@ export class MasterProcess extends EventEmitter {
             throw new Error(message);
         }
         this.serverWorker = serverWorker;
+        this.isRestarting = false;
```

Comment:
please use do it in `final` closure.

Reply:
Sorry, I don't get your idea...

Reply:
it should be in `final` closure to make sure that a flag is restored even if there are some exceptions, although if there is a solution not touching `restart` method i would prefer it